### DISCUSSION TITLE
Rad Pro: Data communications questions

### DISCUSSION THREAD

**Discussion by alfmck** [Gissio/radpro#108] [Ideas]
I've red about renewed Communication protocol Requests. Would like to ask about two of them:
1. **Get tube life pulse count**. The value is updated continuously - does it mean that count updates immediately when the pulse comes?
2. **Get tube rate value** - will the instant rate be calculated using instantaneous rate averaging algorithm as in the devices? 

 Some of Communication protocol Requests are in radpro-tool. Would be very nice if radpro-tool could execute renewed Requests. Radpro-tool would be great single py program for PC controlled radpro systems.

**Comment by Gissio** [comment]
> 1. **Get tube life pulse count**. The value is updated continuously - does it mean that count updates immediately when the pulse comes?

Correct.

> 2. **Get tube rate value** - will the instant rate be calculated using instantaneous rate averaging algorithm as in the devices?

It is the value displayed in the UI, correct.

> Some of Communication protocol Requests are in radpro-tool. Would be very nice if radpro-tool could execute renewed Requests. Radpro-tool would be great single py program for PC controlled radpro systems.

Learn Python, write your own program and publish it on GitHub!
